platform/darwin/src/MGLRasterSource.mm

```diff
@@ -15,88 +17,77 @@ - (instancetype)initWithRawSource:(mbgl::style::RasterSource *)rawSource NS_DESI
 
 @end
 
-@implementation MGLRasterSource
-{
+@implementation MGLRasterSource {
     std::unique_ptr<mbgl::style::RasterSource> _pendingSource;
 }
 
-- (instancetype)initWithIdentifier:(NSString *)identifier URL:(NSURL *)url tileSize:(CGFloat)tileSize
-{
-    if (self = [super initWithIdentifier:identifier]) {
-        _URL = url;
-        _tileSize = tileSize;
-        [self commonInit];
-    }
-    return self;
+- (instancetype)initWithIdentifier:(NSString *)identifier configurationURL:(NSURL *)configurationURL {
+    return [self initWithIdentifier:identifier configurationURL:configurationURL tileSize:0];
```

Comment:
> the raster layer is not visible because of the value 0 here

This demo explicitly applies an opacity function to the layer so that it’s invisible at low zoom levels but becomes visible as you zoom in. The initializer appears to work fine for me.

@jfirebaugh, is it intentional that, by default, we load Retina tiles on non-Retina displays like iPad 2 (due to the default tile size of 512 pixels)?